ONSdigital/dp-healthcheck
# Initial creation of health check library

### What

The initial creation of the DP Health check library, this is an early PR tests are coming. Use this as an opportunity to be as pedantic as possible and ensure the state of the architecture is perfect.

### How to review

Go through the code with a fine tooth comb
- Ensure Go standards are met
- Ensure DP standards are met
- Ensure functionality is accurate
- Check for edge cases 
- Check for spelling
- Check for redundancies or improvements
- Check that the Health check spec requirements are met
- Ensure everything is easily readable
### Who can review

Describe who worked on the changes, so that other people can review.
